### DISCUSSION TITLE
Missing/invalid DB name

### DISCUSSION THREAD

**Discussion by mielune** [Budibase/budibase#5214] [Help] [answered]
Hi folks,

I installed a fresh docker compose using all the default values from .env and docker-compose.yml and loading the default page /builder, I have an 500 for the call `https://budibase.home.lcl/api/global/self` with this body:
{"message":"Missing/invalid DB name","status":500}

I tried to call the couchdb url inside the "app" container with curl and it work fine.

The call to https://budibase.home.lcl/api/global/configs/checklist?tenantId=default  did the same.

Variable for DB is `COUCH_DB_URL: http://${COUCH_DB_USER}:${COUCH_DB_PASSWORD}@couchdb-service:5984`

And no errors during all services startup.

Thanks for your help :)

**Comment by mielune** [comment, marked as answer]
At app startup I can see some calls to CouchDB :

```
  | Apr 1, 2022 @ 17:39:51.618 | [notice] 2022-04-01T15:39:51.617799Z couchdb@127.0.0.1 <0.20393.0> cd5e9bb1b2 couchdb:5984 192.168.37.36 XvpAj7yywASYVswEOniRD4pFxgULsDmV GET /_all_dbs 200 ok 2
  | Apr 1, 2022 @ 17:39:51.610 | [notice] 2022-04-01T15:39:51.610606Z couchdb@127.0.0.1 <0.20392.0> 942e2e0b1e couchdb:5984 192.168.37.36 XvpAj7yywASYVswEOniRD4pFxgULsDmV GET /pouch__all_dbs__/db_global-db? 200 ok 1
  | Apr 1, 2022 @ 17:39:51.610 | [notice] 2022-04-01T15:39:51.610606Z couchdb@127.0.0.1 <0.20391.0> c9abb8c468 couchdb:5984 192.168.37.36 XvpAj7yywASYVswEOniRD4pFxgULsDmV GET /global-db/migrations? 200 ok 1
  | Apr 1, 2022 @ 17:39:51.550 | [notice] 2022-04-01T15:39:51.550713Z couchdb@127.0.0.1 <0.20383.0> 4b6516feae couchdb:5984 192.168.37.36 XvpAj7yywASYVswEOniRD4pFxgULsDmV GET /global-db/ 200 ok 2
  | Apr 1, 2022 @ 17:39:51.550 | [notice] 2022-04-01T15:39:51.550208Z couchdb@127.0.0.1 <0.20384.0> 8787bfef0f couchdb:5984 192.168.37.36 XvpAj7yywASYVswEOniRD4pFxgULsDmV GET /pouch__all_dbs__/db_global-db? 200 ok 1
  | Apr 1, 2022 @ 17:39:51.541 | [notice] 2022-04-01T15:39:51.540918Z couchdb@127.0.0.1 <0.20381.0> 3bc07729bc couchdb:5984 192.168.37.36 XvpAj7yywASYVswEOniRD4pFxgULsDmV GET /pouch__all_dbs__/ 200 ok 4
  | Apr 1, 2022 @ 17:39:51.540 | [notice] 2022-04-01T15:39:51.540153Z couchdb@127.0.0.1 <0.20382.0> 8655ec82d6 couchdb:5984 192.168.37.36 XvpAj7yywASYVswEOniRD4pFxgULsDmV GET /global-db/migrations? 200 ok 1
  | Apr 1, 2022 @ 17:39:51.522 | [notice] 2022-04-01T15:39:51.522345Z couchdb@127.0.0.1 <0.20380.0> ad34f539ef couchdb:5984 192.168.37.36 XvpAj7yywASYVswEOniRD4pFxgULsDmV GET /global-db/ 200 ok 4
```

**Reply by melohagan** [reply]
Hey @mielune Would you be able to share the logs for bbworker whenever you get this error?

**Reply by mielune** [reply]
@melohagan thank's for your kick response, I was offline for the weekend ;)

For the worker, no specific log found, only startup of it, each time I deploy/redeploy the stack : 

```
Apr 1, 2022 @ 17:39:53.267 | Worker running on {"address":"::","family":"IPv6","port":4003}
Apr 1, 2022 @ 17:39:52.254 | $ node dist/index.js
Apr 1, 2022 @ 17:39:52.196 | yarn run v1.22.17
```

I will send my docker-compose here tonight or tomorrow.

**Reply by mielune** [reply]
My trouble was linked to the revers-proxy configuration that was not defined with the good services names inside the stack (I renamed the services without -service suffix).
